src/profile-proxies/profile-proxies.db.ts

```diff
@@ -0,0 +1,109 @@
+import { PROFILE_PROXIES_TABLE } from '../constants';
+import { ProfileProxyEntity } from '../entities/IProfileProxy';
+import {
+  ConnectionWrapper,
+  dbSupplier,
+  LazyDbAccessCompatibleService
+} from '../sql-executor';
+
+export class ProfileProxiesDb extends LazyDbAccessCompatibleService {
+  async insertProfileProxy({
+    profileProxy,
+    connection
+  }: {
+    readonly profileProxy: ProfileProxyEntity;
```

Comment:
I would change it to `Omit<ProfileProxyEntity, 'id' | 'created_at'>` and create those two fields inside this function, not in the service. It just doesn't make sense to let external classes decide those fields.